Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -70,17 +71,10 @@ public function getDescription()
      *
      * @return array($code, $context_counter)
      */
-    private function injectAlignmentPlaceholders(Tokens $tokens, $startAt = null, $endAt = null)
+    private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
     {
-        if (empty($startAt)) {
-            $startAt = 0;
-        }
-
-        if (empty($endAt)) {
-            $endAt = count($tokens);
-        }
-
         for ($index = $startAt; $index < $endAt; ++$index) {
+            /** @var Token $token */
```

Comment:
Sadly that line is missing on 1.10.
I'll remove this hint at the end when this PR is stable